sdk/storage/src/core/into_azure_path.rs

```diff
@@ -72,38 +72,38 @@ mod test {
     }
 
     #[test]
-    #[should_panic(expected = "PathSeparatorNotFoundError")]
     fn no_slash() {
         let path = "containerblob";
         let p = &path;
-        assert!(p.container_name().unwrap() == "container");
-        assert!(path.blob_name().unwrap() == "blob");
+        let _e = AzurePathParseError::PathSeparatorNotFoundError;
```

Comment:
Why `_e`? usually underscores are used to indicate the variable is not used.

Reply:
Fixed